### ISSUE TITLE
Hide query description in dashboard widgets?

### ISSUE THREAD

**Issue by arikfr** [getredash/redash#2431]
As an extension of #1511, maybe we should hide the query description on dashboard widgets? This will give an outlet for users to describe their queries, without cluttering their dashboards. An alternative might be to make it optional (whether to show it or not).

**Comment by arikfr** [comment]
@kravets-levko, @kocsmy wdyt?

**Comment by kocsmy** [comment]
@arikfr absolutely, I think hiding it is a good idea and we can make it optional on widget settings modal.

**Comment by triespe** [comment]
Hi, I would like to express support for more flexibility in dashboard widget titles. 

Query names are usually useful to dashboard developers, while being potentially confusing for dashboard consumers/users so maybe it should be possible to turn it off from dashboard widget titles.

**Comment by P4uton** [comment]
How is the status on this? I can't believe that it is impossible to hide the query name in 2025.

**Comment by lkraider** [comment]
Would like to see this too, the widgets themselves already even have flags to disable display of title/description, the dashboard could receive these same keys and forward them to the widgets.
